### PR TITLE
Docs: development process

### PR DESCRIPTION
**Pull request by robyww**
Development process doc

### CODE REVIEW

File: docs/development-process.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - This example shows the `dev` branch, if you are working on a rc branch always use the rc branch instead of `dev`
+    - This example development branch will be `firefly-123456-table-sort`
+- Branch naming scheme 
+    - Branch Scheme: `Ticket Name` - `word or two description`
+    - Such As: Ticket fix as a table feature might be: `Firefly-123456-table-sort`
+
+### Process
+1. Create a branch using the naming scheme above from the `dev` branch
+    - Create the branch from the base branch
+        - goto base branch: `git checkout dev`
```
Comment (loitly) [inline docs/development-process.md:13]: I normally also do a `git pull` here to ensure I start with the latest as well.

File: docs/development-process.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        - `git pull origin dev`
+        - `git checkout firefly-123456-table-sort`
+    - Rebase
+        - `git rebase -i dev`
+        - While rebasing squash the commit, the `git rebase -i` will put you in and editor to select the commit to squash
+    - Force push your rebased branch
+        - `git push -f origin firefly-123456-table-sort`
+    - goto github
+    - do the basic merge
+    - modify message to start with check ex- `Firefly-123456: merge pull request...`
```
Comment (loitly) [inline docs/development-process.md:44]: `start with check ex-`?  Do you mean to say, `start with ticket name follow by comments, i.e. Firefly-123456: merge...`

**Review by loitly — APPROVED** [review]
Looks good.

File: docs/development-process.md (hunk 2 above)
Comment (robyww) [reply docs/development-process.md:44]: yes, thanks

File: docs/development-process.md (hunk 1 above)
Comment (robyww) [reply docs/development-process.md:13]: yes, I should add that.

File: docs/development-process.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+## Firefly Development process
+
+- _Note on examples_
+    - This example shows the `dev` branch, if you are working on a rc branch always use the rc branch instead of `dev`
+    - This example development branch will be `firefly-123456-table-sort`
+- Branch naming scheme 
+    - Branch Scheme: `Ticket Name` - `word or two description`
+    - Such As: Ticket fix as a table feature might be: `Firefly-123456-table-sort`
+
```
Comment (ejoliet) [inline docs/development-process.md:9]: Remember branch name should start with project in uppercase so it can be integrated in Jira ;-)

**Review by ejoliet — APPROVED** [review]

File: docs/development-process.md (hunk 3 above)
Comment (robyww) [reply docs/development-process.md:9]: I forget. I will fix.

**Pull request merged**